packages/hydrooj/src/handler/misc.ts

```
@@ -109,10 +107,9 @@ export class StorageHandler extends Handler {
     @param('filename', Types.Filename, true)
     @param('expire', Types.UnsignedInt)
     @param('secret', Types.String)
-    async get(domainId: string, target: string, filename = '', expire: number, secret: string) {
-        const expected = md5(`${target}/${expire}/${builtinConfig.file.secret}`);
+    async get({ }, target: string, filename = '', expire: number, secret: string) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)